### PR TITLE
Update Tutorial

### PR DESCRIPTION
**Pull request by tomfrenken**
# Changes

Now leverages the Tekton CLI to give better feedback to the user.
Includes `secret`, `ServiceAccount`, `clusterrole`,`clusterrolebinding` so that you can actually finish the Tutorial without confusion.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

**Comment by tomfrenken** [comment]
Found a few more typos, sry for removing lgtm

File: docs/tutorial.md [hunk 1]
```diff
@@ -152,7 +148,7 @@ spec:
 
 The following is a `Task` with inputs and outputs. The input resource is a
 GitHub repository and the output is the image produced from that source. The
-args of the task command support variable substitution so that the definition of task is
+args of the `Task` command supports variable substitution so that the definition of `Task` is
```
Comment (sbwsg) [inline docs/tutorial.md]: `The args ... support` is correct I think?

File: docs/tutorial.md [hunk 2]
```diff
@@ -255,77 +286,58 @@ tasks/build-docker-image-from-git-source   7m
...
+
+```bash
+tkn taskrun logs build-docker-image-from-git-source-task-run
+```
+
 ## Pipeline
 
-A [`Pipeline`](pipelines.md) defines a list of tasks to execute in order, while
-also indicating if any outputs should be used as inputs of a following task by
+A [`Pipeline`](pipelines.md) defines a list of `Tasks to execute in order, while
```
Comment (sbwsg) [inline docs/tutorial.md]: Looks like you might have missed a backtick after Tasks here.

File: docs/tutorial.md (hunk 1 above)
Comment (tomfrenken) [reply docs/tutorial.md]: Oh true

File: docs/tutorial.md (hunk 2 above)
Comment (tomfrenken) [reply docs/tutorial.md]: good catch

File: docs/tutorial.md [hunk 3]
```diff
@@ -61,61 +63,55 @@ kubectl apply -f <name-of-file.yaml>
...
+Output Resources
+No resources
+
+Params
+No params
+
+Steps
+NAME
+echo
+```
```
Comment (bobcatfish) [inline docs/tutorial.md]: 😍

**Comment by bobcatfish** [comment]
```
I1025 19:35:16.789] level=warning msg="[runner] Can't run linter goanalysis_metalinter: ctrlflow: failed prerequisites: inspect@github.com/tektoncd/pipeline/pkg/artifacts [github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1.test]"
I1025 19:35:16.806] level=error msg="Timeout exceeded: try increase it by passing --timeout option"
```

#1205 maybe?

File: docs/tutorial.md (hunk 3 above)
Comment (tomfrenken) [reply docs/tutorial.md]: Love to hear that haha

**Pull request merged**